managed/services/management/rds.go

```diff
@@ -180,6 +180,13 @@ func (s *RDSService) DiscoverRDS(ctx context.Context, req *managementpb.Discover
 			for _, db := range regInstances {
 				l.Debugf("Discovered instance: %+v", db)
 
+				// This happens when the database is in "Creating" state.
+				// At this point there is no endpoint available.
+				if db.Endpoint == nil {
+					l.Debugf("Instance %q not ready yet. Please wait until the database is fully created in AWS.", *db.DBInstanceIdentifier)
+					continue
+				}
+
```

Comment:
May be we should notify user somehow that instance is present but not ready yet?

Reply:
I think that's a good idea. Maybe we can do it in a new ticket just to make sure we don't panic when we encounter db in this state?

Reply:
Yep, can you please create ticket for that?